db-shared/src/main/java/com/splicemachine/db/iapi/reference/Property.java

```diff
@@ -1291,7 +1291,12 @@ Default value for wait timeouts (60 seconds)
     String ENTERPRISE_ENABLE = "splicemachine.enterprise.enable";
 
     String SPLICE_OLAP_PARALLEL_PARTITIONS = "splice.olapParallelPartitions";
-
+  
     String JOIN_COST_ESTIMATION_MODEL = "joinCostEstimationModel";
+
+    /** The property is used to get the default value for default frequency sketch size of a column.
+     */
+    String FREQUENCY_SKETCH_SIZE = "splice.statistics.defaultFrequencySketchSize";
+    int DEFAULT_FREQUENCY_SKETCH_SIZE = 1024;
```

Comment:
Current values should be seen here: `com/splicemachine/db/iapi/types/DataType.java`, `getQuantilesSketch()`, `getFrequenciesSketch`, and `getThetaSketch`. I think some tests would fail if we actually use 1024 instead of 256 for now.

As far as I understand, this change simply add this property but it's not actually plugged into the system, right?

One question we need to answer is that we have three sketches, each of which has a size. Since we have a single database property for sketch size, which sketch does it control? Or it controls all three?